website/events/templates/events/email/registration_conf_email.html

```diff
@@ -0,0 +1,8 @@
+{% extends "email/html_email.html" %}
+{% block content %}
+<p>Dear {{ registration.member.profile.display_name }},</p>
```

Comment:
```suggestion
<p>Dear {{ name }},</p>
```

Since member may be None, probably easiest is to determine a name outside of the email template and pass it as context.